src/test/java/com/epam/tc/hw4/ex1/ExerciseOneTest.java

```diff
@@ -15,7 +15,6 @@
     @Description("test function: login, correct display icons and text")
     @Feature("base function")
     @Story("perform login and correct page load")
-
```

Comment:
This change should not be a part of MR

Reply:
Not fixed and no answer

Reply:
I deleted this file from mr

Reply:
Yes, it is in history of commit, but not in this branch